## This is to show the exception error were even though may be correct it still causes error to occur

print(10 * (1/0))